.github/workflows/ci.yml

```diff
@@ -22,11 +22,16 @@ jobs:
       - uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # ratchet:actions/setup-python@v5
         with:
           python-version: ${{ env.DEFAULT_PY_VERSION }}
-      - name: Install Linter
+      - name: Install uv
+        id: setup-uv-lint
+        uses: astral-sh/setup-uv@d4b2f3b6ecc6e67c4457f6d3e41ec42d3d0fcb86 # ratchet:astral-sh/setup-uv@v5
+        with:
+          version: ${{ env.UV_VERSION }}
+          enable-cache: true
+          cache-suffix: "linter"
```

Comment:
afaict from the [setup-uv docs](https://github.com/astral-sh/setup-uv?tab=readme-ov-file#should-i-include-the-resolution-strategy-in-the-cache-key), we can leave `cache-suffix` out here and reuse the same cache across all of our jobs. because they're coming from a centrally locked set of deps and we don't vary things like resolution strategies that would cause different jobs to make conflicting package choices.

...that said, leaving this in doesn't seem particularly harmful so maybe we just leave it as-is :shrug: